core/arch/aarch64/proc.c

```diff
@@ -45,6 +45,9 @@ proc_init_arch(void)
     num_simd_registers = MCXT_NUM_SIMD_SLOTS;
     num_opmask_registers = MCXT_NUM_OPMASK_SLOTS;
 
+    set_cache_line_size_using_ctr_el0(/* dcache_line_size= */ &cache_line_size,
```

Comment:
Also, why have "using_ctr_el0" in the name?  I don't think the caller should care about the implementation: that should be hidden behind an abstraction barrier.

Reply:
Modified the name.